stack.yml

```diff
@@ -16,7 +16,7 @@ secrets:
 services:
   ci:
     image: ocurrent/ocaml-ci-service:live
-    command: --github-app-id 39151 --github-private-key-file /run/secrets/ocaml-ci-github-key --github-account-allowlist "talex5,ocurrent,ocaml,mirage,avsm,samoht,kit-ty-kate,tarides,aantron,ocamllabs,realworldocaml,NathanReb,0install,gpetiot,ocaml-ppx,CraigFe,pascutto,julow,ocaml-gospel,vbmithr,gs0510,magnuss,dune-universe,janestreet,emillon,capnproto,ocaml-opam,ocaml-dune,favonia,joelburget,jeffa5,bikallem,jonludlam,g2p,stedolan,ocsigen,dinosaure,hannesm,mirleft,roburio,misterda,ocaml-multicore,cdaringe,inhabitedtype,tmcgilchrist,ocaml-doc,grievejia,Leonidas-from-XIV,ocaml-community,verbosemode,tomjridge,thizanne,n-osborne,TheLortex,patricoferris,routineco,moby,djs55,hyunha,hyper-systems,coco33920,sanette,maiste,yomimono,c-cube,novemberkilo" --confirm above-average --confirm-auto-release 120 --capnp-address=tcp:ci.ocamllabs.io:8102 --github-oauth /run/secrets/ocaml-ci-oauth --submission-service /run/secrets/ocaml-ci-submission.cap --matrix-passfile /run/secrets/matrix-pass --matrix-user ocaml-ci --matrix-host=matrix.ocamllabs.io --github-webhook-secret-file /run/secrets/ocaml-ci-webhook-secret
+    command: --github-app-id 39151 --github-private-key-file /run/secrets/ocaml-ci-github-key --github-account-allowlist "talex5,ocurrent,ocaml,mirage,avsm,samoht,kit-ty-kate,tarides,aantron,ocamllabs,realworldocaml,NathanReb,0install,gpetiot,ocaml-ppx,CraigFe,pascutto,julow,ocaml-gospel,vbmithr,gs0510,magnuss,dune-universe,janestreet,emillon,capnproto,ocaml-opam,ocaml-dune,favonia,joelburget,jeffa5,bikallem,jonludlam,g2p,stedolan,ocsigen,dinosaure,hannesm,mirleft,roburio,misterda,ocaml-multicore,cdaringe,inhabitedtype,tmcgilchrist,ocaml-doc,grievejia,Leonidas-from-XIV,ocaml-community,verbosemode,tomjridge,thizanne,n-osborne,TheLortex,patricoferris,routineco,moby,djs55,hyunha,hyper-systems,coco33920,sanette,maiste,yomimono,c-cube,novemberkilo" --confirm above-average --confirm-auto-release 120 --capnp-public-address=tcp:ci.ocamllabs.io:8102 --capnp-listen-address=tcp:0.0.0.0:9000 --github-oauth /run/secrets/ocaml-ci-oauth --submission-service /run/secrets/ocaml-ci-submission.cap --matrix-passfile /run/secrets/matrix-pass --matrix-user ocaml-ci --matrix-host=matrix.ocamllabs.io --github-webhook-secret-file /run/secrets/ocaml-ci-webhook-secret
```

Comment:
This shouldn't need to change, since we port map 8102:9000 [here in the stack.yml](https://github.com/ocurrent/ocaml-ci/pull/460/files#diff-85060223500160bc2e4c8246e65790c0b04e1fd64a49d8f2cac78a89e80f46c1R25)